routemaster/feeds.py

```diff
@@ -0,0 +1,51 @@
+"""Creation and fetching of feed data."""
+from typing import Dict
+
+import requests
+
+from routemaster.utils import get_path
+
+
+def feeds_for_state_machine(state_machine) -> Dict[str, 'Feed']:
```

Comment:
Minor: should the argument by type-annotated? (I'm not sure how much typing coverage we're aiming for)

Reply:
We actually can't here due to https://github.com/python/mypy/issues/4049 – there's detail in the commit message that removed the type.